packages/asset-kit-block/src/AssetKitBlock.tsx

```diff
@@ -92,10 +109,16 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
                                 status
                             ) || currentAssets.length === 0
                         }
-                        onClick={startDownload}
+                        onClick={isEditing ? undefined : startDownload}
                         style={BlockStyles.buttonPrimary}
```

Comment:
Hover isn't working, check the serializer of the ButtonNode to get a clue how it will work

Reply:
Will be fixed in a separate PR

Reply:
Own task for it